newIDE/app/src/ObjectEditor/ObjectEditorDialog.js

```diff
@@ -68,6 +80,20 @@ export class ObjectEditorDialog extends Component<*, StateType> {
         }
         titleStyle={styles.titleContainer}
       >
+        <div style={styles.container}>
+          Object Name:
+          <SemiControlledTextField
+            style={styles.objectTitle}
```

Comment:
I think we can do what we want without any styling. Try something like:
```
<Line>
  <Column>
    Object Name: 
  </Column>
  <Column expand>
    <SemiControlledTextField ... fullWidth ... />
  </Column>
</Line>
```

Reply:
done..Looks good, but "Object Name" comes up too high
![capture](https://user-images.githubusercontent.com/6495061/42350204-5e218c46-80a7-11e8-9887-102168330b9b.PNG)

Reply:
Can you try with `<Line alignItems="baseline">`? :)